### ISSUE TITLE
Add Java CFEnv To Spring Initializr

### ISSUE THREAD

**Issue by making** [pivotal-cf/java-cfenv#76]
It would be nice to have java-cfenv in Spring Initializr as the current option for CF is Spring Cloud Connector which has been [deprecated](https://github.com/spring-projects/spring-boot/issues/17877).

The addition of a project seems to need to be proposed by the project's maintainers via a completed [new entry proposal](https://github.com/spring-io/start.spring.io/issues/new?template=new-entry-proposal.md).

**Comment by dyroberts** [comment]
Thanks @making. We are actively working on making this happen. ;)

**Comment by dyroberts** [comment]
Blocked by cloudfoundry/java-buildpack#779

**Comment by making** [comment]
any update?
As of cf-for-k8s 1.0, it  doesn't include Spring AutoReconfiguration, so java-cfenv is necessary to bind a service out of the box.
Specify the version of java-cfenv every time is painful.

**Comment by pivotal-david-osullivan** [comment]
As per the latest update https://github.com/cloudfoundry/java-buildpack/issues/779#issuecomment-1084682829, this is no longer blocked on the JBP
